modules/admin_manual/pages/installation/manual_installation/server_prep_ubuntu_20.04.adoc

```diff
@@ -16,24 +16,15 @@
 
 == Introduction
 
-The target audience for this document are more sophisticated admins with additional needs and
-setup scenarios. Ubuntu 20.04 default only provides PHP 7.4. In case you are running a PHP
-version lower than 7.4, your PHP version is either already EOL or, for PHP 7.3, it will be EOL
-in 2021. Consider upgrading to PHP 7.4. For more information see
-{php-supported-versions-url}[Currently Supported PHP Versions]. This document considers two basic scenarios.
-You have/plan to upgrade from 18.04 LTS to 20.04 LTS or you have a fresh Ubuntu 20.04 LTS installation.
+The target audience for this document are more sophisticated admins with additional needs and setup scenarios. Ubuntu 20.04 default only provides PHP 7.4. In case you are running a PHP version lower than 7.4, your PHP version is either already EOL or, for PHP 7.3, it will be EOL in 2021. Consider upgrading to PHP 7.4. For more information see {php-supported-versions-url}[Currently Supported PHP Versions]. This document considers two basic scenarios. You have/plan to upgrade from 18.04 LTS to 20.04 LTS or you have a fresh Ubuntu 20.04 LTS installation.
```

Comment:
```suggestion
The target audience for this document are experienced admins with additional needs and setup scenarios. Ubuntu 20.04 by default only provides PHP 7.4. In case you are running a PHP version lower than 7.4, your PHP version is already EOL. Consider upgrading to PHP 7.4. For more information see {php-supported-versions-url}[Currently Supported PHP Versions]. This document considers two basic scenarios. You have/plan to upgrade from 18.04 LTS to 20.04 LTS or you have a fresh Ubuntu 20.04 LTS installation.
```